### DISCUSSION TITLE
Scaling

### DISCUSSION THREAD

**Discussion by avin-kavish** [breejs/bree#183] [General] [answered]
Is bree scalable to multi node clusters? I think not having a persistance layer is going to limit it to a single node.

**Comment by shadowgate15** [comment]
You can setup your own persistance layer using plugins. This is something we have wanted to work on but have not had time to put it together yet. However you can use databases inside of a job to get information from a database, lock it and perform any actions, necessary.

**Comment by titanism** [comment, marked as answer]
Correct, if you do not have a persistence layer (e.g. bring your own database) then you will only be able to run one process if your jobs could have read/write/lock/duplicate issues.
